runtime/runtime.go

```diff
@@ -0,0 +1,116 @@
+package runtime
+
+import (
+	"log"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/pokt-network/pocket/consensus/types"
```

Comment:
Let's be consistent with import naming `consensus/types` to `consTypes`

Reply:
Addressed in https://github.com/pokt-network/pocket/pull/235/commits/a8530d2ab589c248d53352a8df2b37d351b362ac